firefox_decrypt.py

```diff
@@ -409,7 +410,7 @@ def decode_entry(self, user64, passw64):
 
         return user, passw
 
-    def decrypt_passwords(self, export, tabular):
+    def decrypt_passwords(self, export, format="human", csv_delimiter=";", csv_quotechar="|"):
```

Comment:
Perhaps `output_format` instead of `format`. `format` is actually a function on python 3.